frontend/src/components/Chart.tsx

```diff
@@ -2,12 +2,17 @@ import "../assets/Chart.scss";
 
 import Highcharts from "highcharts";
 import HighchartsReact from "highcharts-react-official";
-import PropTypes from "prop-types";
-import React from "react";
+import * as React from "react";
 
-const Chart = (props) => {
-  const { data, dataSources, campaigns } = props;
+import { Data } from "../types/types";
 
+type Props = {
+  data: Data[];
```

Comment:
Updating this type to `ReadonlyArray<Data>` will fix your error with invalid prop type when calling `<Chart />` in `App`